api/src/main/java/io/strimzi/api/kafka/model/EntityTopicOperatorSpec.java

```diff
@@ -102,12 +103,12 @@ public void setTopicMetadataMaxAttempts(int topicMetadataMaxAttempts) {
     }
 
     @Description("Resource constraints (limits and requests).")
-    public Resources getResources() {
+    public ResourceRequirements getResources() {
```

Comment:
Should we use the `@KubeLink` annotation to link it to the Kubernetes API reference?